Content.Server/Holiday/Christmas/RandomGiftSystem.cs

```diff
@@ -54,6 +57,7 @@ private void OnUseInHand(EntityUid uid, RandomGiftComponent component, UseInHand
 
         var coords = Transform(args.User).Coordinates;
         var handsEnt = Spawn(component.SelectedEntity, coords);
+        _adminLogger.Add(LogType.Action, LogImpact.Low, $"{ToPrettyString(args.User)} used {ToPrettyString(component.Owner)} which spawned {ToPrettyString(handsEnt)}");
```

Comment:
This should use a new LogType, named something like LogType.RandomGift, not LogType.Action